### DISCUSSION TITLE
v2.9.6

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#5087] [分享 / Share]
✨新增
- `自定义顶栏` 更新:
  - (来自 v2.9.5-preview) 历史支持手动刷新. (#4948)
  - 全局固定适配话题页. (#5039)
  - 新歌热榜 URL 更新. (#5056)
  - `订阅` 重命名为 `番剧`, 功能不变. (#4777)
  - 增加 VLOG, 搞笑, 综艺, 单机游戏入口. (#1511)
- `传统连播模式` 支持番剧页面. (#5067)
- `下载视频` 增加选项 `DASH 回退编码`, 默认回退到 AVC, 可以避免 HEVC 回退到 AV1 的情况. (#5080)
- 新增功能 `隐藏首页轮播图`. (从上一版本废弃的 `简化首页` 中提取) (#5038)
- `夜间模式` 优化对动态和新版首页的适配. (#5075)

🐛修复
- 修复 `隐藏头像框` 对部分角标删除不完全. (#5040)
- 修复 `自定义顶栏` 删除搜索历史时失焦. (#4732)
- 修复 `自定义顶栏` 未登录时仍能通过功能面板进行排序. (#5051)
- 修复 `自定义顶栏` 收藏夹的音频链接错误. (#5069)
- 修复在搜索页的样式冲突. (#5073)
- 修复 `直播首页静音` 在关闭时仍会导致首页静音. (#5045)
- 修复 `禁止滚轮调音量` 在 Firefox 中不生效. (#5047)
- 回退 `直播信息扩充` 的 API 变更. (#4964)

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.9.6'>v2.9.6</a>.</em>

**Comment by Geeyun-JY3** [comment]
我这边`自定义顶栏`的`历史`里出现了两个刷新按钮：
![image](https://github.com/user-attachments/assets/1e34c76c-793e-41f8-96aa-4de210e36e32)

**Reply by the1812** [reply]
合码合得有点问题...

**Comment by Geeyun-JY3** [comment]
`自定义顶栏`的`订阅`不能改名为`番剧`啊，别忘了之前做了个番剧入口：#1938
![image](https://github.com/user-attachments/assets/2b9f14cd-3096-4596-aa7e-b3092db00bac)

**Reply by the1812** [reply]
那应该叫啥比较好, 在空间里是叫追番追剧

**Reply by Geeyun-JY3** [reply]
不嫌长的话就跟随官方叫法吧。
